### PR TITLE
Added Spotify Clone Project

### PR DESCRIPTION
**Pull request by SadafKausar2025**
## Related Issue #802 

[Cite any related issue(s) this pull request addresses. If none, simply state "None”]

## Description

[Please include a brief description of the changes or features added]

## Type of PR
- [x] Feature enhancement

## Screenshots / Videos (if applicable)
Screenshot (1963)

https://github.com/CodeHarborHub/codeharborhub.github.io/assets/137484958/7bc66c1e-9da8-4503-8b7b-3c8d71430d28

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

**Comment by SadafKausar2025** [comment]
hello @Ajay-Dhangar 
please review this PR and merge it
Please give levels accordingly . it was good project that I added

File: src/data/userData.tsx [hunk 1]
```diff
@@ -9,6 +9,15 @@ export const Users: User[] = [
     website: "https://my-youtube-ajay.vercel.app/",
     source: "https://github.com/Ajay-Dhangar/youtube_clone",
     tags: ["opensource", "clone", "favorite", "react"],
+  },
+    {
+    title: "Spotify Clone",
+    description:
+      "Build and Deploy a Modern Spotify Clone Application in React JS with Material UI 5",
+    preview: require("./showcase/spotify.png"),
+    website: "https://spotifyclone234.freewebhostmost.com/",
```
Comment (Ajay-Dhangar) [inline src/data/userData.tsx:18]: The current domain (`https://spotifyclone234.freewebhostmost.com/`) is not secure. Could you please replace it with a more secure URL or domain? May be used `github-page`, `vercel`, and some more secure and trustable 

image

File: dsa-solutions/lc-solutions/0000-0099/0017-letter-combinations-of-a-phone-number.md [hunk 2]
```diff
@@ -1,89 +1,104 @@
 ---
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/0000-0099/0017-letter-combinations-of-a-phone-number.md:1]: Please resolve the issues with this site. It seems there may have been some changes made to this file, which are causing problems related to your task. Removing these changes might help fix the issues.

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
@SadafKausar2025 update and change your code based on suggestions

**Comment by Ajay-Dhangar** [comment]
> hello @Ajay-Dhangar please review this PR and merge it Please give levels accordingly . it was good project that I added

The level is provided by GSSOC'24; it is not part of my personal contributions. You added some text as an update, so I only included level 1. Additionally, I assigned every contributor who added their project to the showcase.

**Comment by SadafKausar2025** [comment]
@Ajay-Dhangar 
In Edge the domain https://spotifyclone234.freewebhostmost.com/   is secure . it is opening in my PC.

Screenshot (1967)

**Comment by Ajay-Dhangar** [comment]
> @Ajay-Dhangar In Edge the domain https://spotifyclone234.freewebhostmost.com/ is secure . it is opening in my PC.
> 
> Screenshot (1967)

### In Edge

I can't say whether your project is right or wrong, but for security purposes and based on my site's security policies, it is my responsibility to use secure sources for everything. I have also mentioned this in my documentation and content.

image

**Comment by SadafKausar2025** [comment]
check this 
https://sadafkausar2025.github.io/Spotify-Clone/

**Comment by Ajay-Dhangar** [comment]
@SadafKausar2025, I suggest exploring another secure project for us. If you don't mind, I can review your site code and help you deploy it using GitHub Pages. However, I'll be able to do this after my GSSOC'24 contributions, as I am currently busy.

**Comment by SadafKausar2025** [comment]
ok

**Comment by SadafKausar2025** [comment]
I will try to make secure

**Comment by SadafKausar2025** [comment]
hello @Ajay-Dhangar 
please check now
https://spotifyclone234.freewebhostmost.com/

**Comment by Ajay-Dhangar** [comment]
> https://spotifyclone234.freewebhostmost.com

I believe there might be some unauthorized code in your source, which could be causing the issue you're experiencing.

**Comment by SadafKausar2025** [comment]
Is it working now?

**Comment by Ajay-Dhangar** [comment]
@SadafKausar2025 Apologies for the inconvenience. I'd like to close this particular issue for now, but feel free to focus on other tasks or issues.

**Comment by SadafKausar2025** [comment]
I checked now it is secure  https://spotifyclone234.freewebhostmost.com

**Comment by SadafKausar2025** [comment]
hello @Ajay-Dhangar 
I Have corrected it please open again now it is secure
https://spotifyclone234.freewebhostmost.com/

**Pull request closed without merging**